src/AutoRest.CSharp/LowLevel/Generation/LowLevelExampleComposer.cs

```diff
@@ -566,6 +566,12 @@ private string MockParameterTypeValue(string parameterName, CSharpType parameter
                     return "Guid.NewGuid()";
                 }
 
+                if (type == typeof(WaitUntil))
+                {
+                    // use `Started`, since we will generate `operation.WaitForCompletion()` afterwards
+                    return $"{nameof(WaitUntil)}.{nameof(WaitUntil.Started)}";
+                }
```

Comment:
Well, I suggest it would be better if we could change this huge if into a switch statement.

Reply:
> Well, I suggest it would be better if we could change this huge if into a switch statement.

@ArcturusZhang surprisingly, `typeof` cannot be used in `switch...case`: https://stackoverflow.com/questions/1711005/why-doesnt-c-sharp-switch-statement-allow-using-typeof-gettype